extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -290,41 +290,45 @@
     @ConfigGroup
     public static class RuntimeCatalogConfig {
         /**
-         * Used to control the resolution of components catalog info.
+         * Whether the {@code json} resources under {@code META-INF/services/org/apache/camel/component} should be
```

Comment:
the json is not located at  `META-INF/services/org/apache/camel/component` but the catalog **may** uses service files to determine the class name of the component as the json is usually located in the same package as the component class.

Same for the other descriptions

Reply:
Thanks, I have changed the text in 9affaa1 . Could you please re-review?